drv/transceivers-server/src/main.rs

```diff
@@ -41,7 +44,7 @@ task_slot!(SENSOR, sensor);
 include!(concat!(env!("OUT_DIR"), "/i2c_config.rs"));
 
 #[allow(dead_code)]
-#[derive(Copy, Clone, PartialEq, Eq)]
+#[derive(Copy, Clone, PartialEq, Eq, Count)]
 enum Trace {
     None,
```

Comment:
similarly, we can avoid counting this 
```suggestion
    #[count(skip)]
    None,
```